.github/workflows/CI.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         echo "$missing_tests are missing the close_figures_on_teardown fixture."
         exit 1
         fi
+#   - name: Check version number
+#     run: python ./bin/check_version.py
```

Comment:
This would do that, but this script will only work when `_version.py` is on the master branch, so I will follow this PR with an immediate second PR uncommenting this.